build/build-bundle.js

```diff
@@ -190,7 +190,6 @@ async function buildBundle(entryPath, distPath, opts = {minify: true}) {
         // resolved eventually.
         plugins.partialLoaders.inlineFs({
           verbose: Boolean(process.env.DEBUG),
-          ignorePaths: [require.resolve('puppeteer-core/lib/esm/puppeteer/common/Page.js')],
```

Comment:
`Page.js` has moved but it looks like the comment that contained `fs.readFileSync` is not longer there so we should be good to remove.